docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -19,7 +19,7 @@ The minimal required version of .NET Framework is 4.7.2.
     - Ensure C#, VB, MSBuild, .NET Core and Visual Studio Extensibility are included in the selected work loads
     - Ensure Visual Studio is on Version "RC1" or greater
     - Ensure "Use Previews" is checked in Tools -> Options -> Projects and Solutions -> .NET Core
-1. [.NET Core SDK 3.0 Preview 6](https://dotnet.microsoft.com/download/dotnet-core/3.0) [Windows x64 installer](https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview6-011773/dotnet-sdk-3.0.100-preview6-011773-win-x64.exe )
+1. [.NET Core SDK 3.0 Preview 6](https://dotnet.microsoft.com/download/dotnet-core/3.0) [Windows x64 installer](https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview6-012105/dotnet-sdk-3.0.100-preview6-012105-win-x64.exe )
```

Comment:
Do make sure to email the team when this goes in, because otherwise people won't be aware of it.

Reply:
Can you guys please post this information somewhere, or ping the community?  Ths stuff just ends up breaking external people with *zero* information as to what's going on.  Tagging @jinujoseph  as well on this.

Reply:
@CyrusNajmabadi yes , we will .